README.md

```diff
@@ -1 +1,91 @@
-# friendly-giggle
+# pet-cam-fg
+Pet cam client and API that serves connection to user and their pet. This project is intended to run alongside [pet-cam-aa](https://github.com/wongbros/automatic-adventure).
+
+## Description
+A web application that securely connects a user with their pet over web cam. A user can simply login using their Google account and setup details including a whitelist of phone numbers that are allowed to connect to their room. After starting the camera, only the user with their hashed url, which can be obtained via SMS to ensure privacy, can connect to the camera.

```

Comment:
Change description a little. Say how it's supposed to be the a client that identifies users using the lookup. It also establishes video connection when the secured url is requested.